client/assets/components/navbar.scss

```diff
@@ -0,0 +1,171 @@
+.top-navbar {
+  height: 70px;
+  border-radius: 0;
+  background-color: #fff;
+  margin: 0;
+  border: 0;
+  .navbar-brand {
+    height: 70px;
+    width: 70px;
+    display: flex;
+    align-items: center;
+    justify-content: center;
+  }
+}
+.navbar-collapse.collapse {
+  display: flex !important;
+  height: 70px !important;
+}
+.top-navbar__right-container {
+  align-items: center;
+  justify-content: right;
+  padding: 0;
+  .nav-profile {
+    width: 183.5px;
+    &__list-menus {
+      display: flex;
+      flex-direction: column;
+      align-items: center;
+      box-shadow: -1px 6px 6px #dedede;
```

Comment:
### Nitpick

The color #DEDEDE is going to be used in a lot of places, let's add that definition to the `variables.scss`.